content/docs/operator-guide/create-a-stack.md

```diff
@@ -0,0 +1,327 @@
++++
+title="Create a stack"
+weight=2
++++
+
+Creating a custom [stack][stack] allows you to configure the base images for the build-time environment for your [builder][builder] and the run-time for your application.
+
+<!--more-->
+
+## Prerequisites
+
+Before we get started, make sure you've got the following installed: 
+
+{{< download-button href="https://store.docker.com/search?type=edition&offering=community" color="blue" >}} Install Docker {{</>}}
+
+
+## Creating a custom stack
+
+In this tutorial we will create a sample stack based on `Ubuntu Bionic`. To create a custom stack, we simply need to create customized build and run images. Let's see how we can do so!
+
+
+### Create a common base image
+
+Let's start by creating a base image containing layers that will be required by both the `build` and `run` images. In order to do this, switch to a clean workspace and create a `Dockerfile` as specified below:
+
+#### Defining the base
+We start with choosing `ubuntu:bionic` as our `base` image. Since we will be reusing these layers in both our build and run images we will be defining a common base image and leveraging [Docker's multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) to ensure this acts as the common base image for both our build-time and run-time environment.
```

Comment:
```suggestion
We start with `ubuntu:bionic` as our `base` image. Since we will be reusing these layers in both our build and run images we will be defining a common base image and leveraging [Docker's multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) to ensure this acts as the common base image for both our build-time and run-time environment.
```